package.json

```diff
@@ -19,8 +21,8 @@
     "lint"
   ],
   "scripts": {
-    "start": "nodemon src/index.js",
-    "lint": "semistandard src/index.js",
+    "start": "nodemon src/server/index.js",
```

Comment:
Why did we add this 'server' directory. This is the repo for this server, are we expecting other kinds of src-s. It seems redundant